var/exp_repo/experiments/laghos/experiment.py

```diff
@@ -0,0 +1,62 @@
+from benchpark.directives import variant
+from benchpark.experiment import Experiment
+
+
+class Laghos(Experiment):
+
+    variant(
+        "experiment",
+        default="example",
+        values=("weak", "strong","example"),
+        description="weak or strong scaling",
+    )
+
+    def compute_applications_section(self):
+        variables = {}
+            
+        if self.spec.satisfies("scaling=weak"):
+            variables["rs"] = ["5","6"]
+            variables["n_nodes"] = ["1","8"]
+        elif self.spec.satisfies("scaling=strong"):
+            variables["n_nodes"] = ["1","8"]
+            variables["rs"] = "5"
+        else:
+            variables["n_nodes"] = ["1","2","4","8","16","32","64","128"]    
+        variables["n_ranks"] = "{sys_cores_per_node} * {n_nodes}"
+
+        return {
+            "laghos": {  # ramble Application name
+                "workloads":{
+                    # TODO replace with a hash once we have one?
+                    "problem": {
+                       #"variables": variables,
+                        "experiments": {
+                            "laghos_{n_nodes}_{n_ranks}": {
```

Comment:
Can you add the other parameters into the naming scheme here, so basically anything that gets passed in (rs, rp, etc) should be in here, see the naming for the kripke experiment in https://github.com/LLNL/benchpark/pull/344/files

Reply:
@august-knox I enabled a dry-run test and it is failing.  Please diagnose/fix.